templates/shared/combomarkdowneditor.tmpl

```diff
@@ -19,29 +19,28 @@ Template Attributes:
 	</div>
 	{{end}}
 	<div class="ui tab active" data-tab-panel="markdown-writer">
-		<markdown-toolbar class="gt-df">
+		<markdown-toolbar class="gt-df gt-ac gt-gap-3">
 			<div class="markdown-toolbar-group">
-				<md-header class="markdown-toolbar-button">{{svg "octicon-heading"}}</md-header>
-				<md-bold class="markdown-toolbar-button">{{svg "octicon-bold"}}</md-bold>
-				<md-italic class="markdown-toolbar-button">{{svg "octicon-italic"}}</md-italic>
+				<md-header role="button" class="markdown-toolbar-button btn-link muted" data-tooltip-content="{{.locale.Tr "editor.buttons.heading.tooltip"}}">{{svg "octicon-heading"}}</md-header>
```

Comment:
I really dislike this `btn-link muted`, it just duplicates a lot.

Why can't we put the styles in `.markdown-toolbar-button` ?

Reply:
I won't say blocker for this change, but I will refactor such usage in next PR.

I do not see why we should add a style then mute some of the styles, and I do not think the duplicated class names are maintainable.


As I explained above, we are styling the toolbar buttons, we only need a `.markdown-toolbar-button {}` style for them.